### PR TITLE
add a pull request template for the project

### PR DESCRIPTION
**Pull request by tvieira**
We need our pull request to be standardized so we can better
understand the changes and the rationale behind the changes.

### CODE REVIEW

**Review by smatula — APPROVED** [review]
LGTM

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+(Thanks for sending a pull request! Please make sure you read the contribution
+guidelines [CONTRIBUTING.rst](CONTRIBUTING.rst), then fill out a PR like the
```
Comment (ryankwilliams) [inline .github/PULL_REQUEST_TEMPLATE.md]: This link gives you 404 page not found. I think you need to fix the reference to `CONTRIBUTING.rst` to the following:

`[CONTRIBUTING.rst](../CONTRIBUTING.rst)`

Or you can give a static HTTP reference to the CONTRIBUTING.rst file.

**Review by vi-patel — APPROVED** [review]
looks good.

**Review by ryankwilliams — APPROVED** [review]
LGTM

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+* use the imperative, present tense: "change" not "changed" nor "changes" nor
+  "changing"
+
+* don't capitalize first letter
+
+* no dot (.) at the end
+
+**Notes on body:**
+
+* do not save words in the body. The more information the better to understand
```
Comment (tehsmyers) [inline .github/PULL_REQUEST_TEMPLATE.md]: This language is a little hard for me to parse. Perhaps "* be descriptive about why this change is useful and how it is an improvement -- the more information the better"

**Review by tehsmyers — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (tvieira) [reply .github/PULL_REQUEST_TEMPLATE.md]: Totally agree :) Thanks @seandst !

**Review by tehsmyers — APPROVED** [review]
